### PR TITLE
GDB-7678: Set up a GitHub repository for the "ontotext-yasgui" component

### PR DESCRIPTION
**Pull request by boyan-tonchev**
What?

  An initial commitment of "ontotext-yasgui" projects.

Why?

  We need a repository where to manage customization of yasgui component.

How?

  A repository "ontotext-yasgui" is created. It holds three projects.
  - Yasgui - holds source code of original yasgui-component;
  - ontotext-yasgui - holds ontotext customizations;
  - cypress - holds e2e tests.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
...
+It contains the source code of yasgui. After build of project yasgui.min.js and yasgui.min.css artefacts will
+be copied into "ontotext-yasgui".
+
+## [ontotext-yasgui](ontotext-yasgui/README.md)
+This is main project of "ontotext-yasgui". In this project we will customize yasgui for Ontotext needs.
+
+## [cypress](cypress)
+It contains e2e tests.
+
+# Approach for the component implementation
```
Comment (svilenvelikov) [inline README.md:18]: This should be in the developers-guide. Maybe here we should just link to it here.

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
...
+    - Yasgui/packages/yasgui/node_modules
+    - Yasgui/packages/yasgui/build
+    - Yasgui/packages/yasqe/node_modules
+    - Yasgui/packages/yasqe/build
+    - Yasgui/packages/yasr/node_modules
+    - Yasgui/packages/yasr/build
+After execution of command the projects will be cleared as just cloned from repository.
+> For development purpose the minimization of yasgui can be skipped by setting minimize property to false of optimization object in Yasgui/webpack/config.ts file.   
+
+# Dev server
```
Comment (svilenvelikov) [inline README.md:63]: Change to `Starting a development server`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ontotext-yasgui-web-component",
+  "version": "1.0.0",
```
Comment (svilenvelikov) [inline package.json:3]: Make this `0.0.1`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "ontotext-yasgui-web-component",
```
Comment (svilenvelikov) [inline package.json:2]: Didn't we decided to name it `ontotext-yasgui` and the inner ontotext-yasgui to be `ontotext-yasgui-web-component`.

**Review by svilenvelikov — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (boyan-tonchev) [reply package.json]: done

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**